general-package-development.Rmd

```diff
@@ -18,7 +18,7 @@ If there are any remaining, a justification of why they are not corrected will b
 
 ### BiocCheck
 
-Packages must also pass `BiocCheck::BiocCheckGitClone()` and `BiocCheck::BiocCheck()` with no errors and no warnings.
+Packages must also pass `BiocCheck::BiocCheckGitClone()` and `BiocCheck::BiocCheck('new-package'=TRUE)` with no errors and no warnings.
```

Comment:
@lshep 

sure it's not backticks around `new-package` instead of single quotation?

Reply:
I tested with both backticks and quotes and it worked for me either way but if you find that it doesn't feel free to change